wdl-lint/tests/lints/missing-blank-line-after-version/source.wdl

```diff
@@ -0,0 +1,6 @@
+## This is a test of a missing blank line following the version statement.
+
+version 1.1
```

Comment:
Maybe this is OOS, but where would the following case be caught?
```wdl
version 1.1




import "foo"
```

Reply:
It's caught by the existing `Whitespace` rule:

```
warning[Whitespace]: more than one blank line in a row
>  ┌─ tests/lints/import-placements/source.wdl:5:1
>  │  
>5 │ ╭ 
>6 │ │ 
>7 │ │ 
>8 │ │ 
>9 │ │ import "good"
>  │ ╰^
>  │  
>  = fix: remove the unnecessary blank lines